sqlalchemy_media/stores/gcp.py

```diff
@@ -0,0 +1,94 @@
+""""
+    Google cloud storage
+"""
+from io import BytesIO
+
+from google.cloud import storage
+from google.cloud.exceptions import NotFound
+
+from sqlalchemy_media.exceptions import GCPError
+from sqlalchemy_media.optionals import ensure_gcs
+from .base import Store
+from ..typing_ import FileLike
+
+
+class GCPStore(Store):
```

Comment:
@ribrea GCP Means Google Cloud Platform. I think  the more accurate term is GoogleCloudStorge (apparently GCS is not a valid term too)

Reply:
that's alright i just took the name from task title.
Fixes #1f98f21